src/IssueViz.UnitTests/Editor/ErrorTagging/ErrorTagTooltipProviderTests.cs

```diff
@@ -71,6 +71,8 @@ public void Create_CreatesTooltipWithHyperlink()
             var hyperlink = (Hyperlink)inlines[0];
             hyperlink.Command.Should().Be(navigateCommand);
             hyperlink.CommandParameter.Should().Be("some rule");
+
+            hyperlink.TextDecorations.Should().BeNull(); // no decorations by default i.e. not underlined
```

Comment:
We can't easily check the mouse-over behaviour, but we can at least check that the tooltip is not underlined by default.